examples/applications/plot_species_distribution_modeling.py

```diff
@@ -31,7 +31,7 @@
 ----------
 
  * `"Maximum entropy modeling of species geographic distributions"
-   <http://www.cs.princeton.edu/~schapire/papers/ecolmod.pdf>`_
```

Comment:
Yeah but given that http://www.cs.princeton.edu/~schapire/ says:


> I have moved to Microsoft Research in NYC.
My new website is: rob.schapire.net. 

I would be enclined to change all the links.